Pipfile

```diff
@@ -4,7 +4,9 @@ verify_ssl = true
 name = "pypi"
 
 [packages] # Packages required to run the application
-microsoft-kiota-http = "*"
+microsoft-kiota-abstractions = '*'
+microsoft-kiota-http = "==0.1.2"
+microsoft-kiota-authentication-azure = '*'
```

Comment:
we should probably lock the major here to avoid any surprises

Reply:
Updated in https://github.com/microsoftgraph/msgraph-sdk-python-core/pull/141/commits/7cd67e4635bbd783fb75b110cc2519f3b0c6ac97